internal/pkg/service/cli/dialog/init.go

```diff
@@ -56,8 +57,22 @@ func (p *Dialogs) AskInitOptions(ctx context.Context, d initDeps) (initOp.Option
 	}
 
 	// Ask for workflows options
-	if p.Confirm(&prompt.Confirm{Label: "Generate workflows files for GitHub Actions?", Default: true}) {
-		out.Workflows = p.AskWorkflowsOptions(d.Options())
+	options := d.Options()
+	if options.IsSet("ci") && !options.GetBool("ci") {
+		if options.IsSet("ci-validate") || options.IsSet("ci-push") || options.IsSet("ci-pull") {
+			return out, errors.New("`ci-*` flags may not be set if `ci` is set to `false`")
+		}
+
+		out.Workflows = workflowsGen.Options{
+			Validate:   false,
+			Push:       false,
+			Pull:       false,
+			MainBranch: options.GetString("ci-main-branch"),
+		}
+	} else {
```

Comment:
Tohle ale znamená, že když je `--ci=true`, tak spustí interaktivní dotaz. Nemělo by to být teda tak, že:
- `--ci=false` - negeneruje CI
- `--ci=true` - generuje CI
- a až při chybějícím flagu se ptá? (Tj. taky by neměl mít default.)

Reply:
Bylo by dobrý kdyby k tomu byl test, `init_test.go` tu už i existuje.

Reply:
Ok, teď je to teda:
- not set -> ptá se
- `--ci=true` -> generuje všechno
- `--ci-false` -> negeneruje nic